src/jobs/webhooker.ts

```
@@ -4,7 +4,9 @@ import { amqpEnqueue } from '../amqp'
 import { UNOAPI_DELAY_AFTER_FIRST_MESSAGE_WEBHOOK_MS, UNOAPI_JOB_WEBHOOKER } from '../defaults'
 import { extractDestinyPhone } from '../services/transformer'
 import logger from '../services/logger'
-const  delays: Map<String, boolean> = new Map()
+
+const  dUntil: Map<String, number> = new Map()
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)